modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -108,23 +121,32 @@ curl "http://localhost/myResource" -u 1234:abcd
 
 Example DataWeave 2.0 expression to be used when configuring the policy:
 
-No DataWeave 2.0 expression needed to be specified when the policy is configured to use the Basic Authentication as the origin of the credentials.
+To use the Basic Authentication as the origin of the credentials, no DataWeave 2.0 expression needs to be specified when configuring the policy.
+
+=== Configure API Specifications
+
+The Client ID Enforcement policy requires changes in your API specification to implement the credentials requirement.
+You can find the RAML or OAS snippet link containing the RAML or OAS code you need to add to the API specification in the corresponding policy. You can access this code from the list of applied policies in the Policies tab of your API specification in API Manager.
+//future enhancement: how to update API spec once a policy is applied.
+
+=== Policy Output
 
-== WWW Authenticate HTTP header
-When a HTTP request is performed against a protected resource, where the Client ID enforcement policy is applied, and client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
+When an HTTP request is performed against a protected resource for which the Client ID Enforcement policy is applied, and the client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
 
 [width="100%", cols="2,70"]
 |===
 >s| Custom mode | Header 'WWW-Authenticate'='Client-ID-Enforcement'
 >s| Basic Auth mode | Header 'WWW-Authenticate'='Basic realm="mule-realm"
 |===
 
+For SOAP applications, the response will be a SOAP error.
+
 == See Also
 
+* xref:manage-client-apps-latest-task.adoc[Managing API instance Contracts on API Manager]
```

Comment:
Encryption link is missing. Might be also good to add a link to the release notes on each policy, what do you think?

Reply:
Good idea! I'll add for release notes too.